Dockerfile.redhat

```diff
@@ -66,9 +66,9 @@ RUN dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.n
             which \
             yum-utils \
             unzip \
+            vim \
```

Comment:
Is this needed ?

Reply:
It's only for the build image. It's useful to have it there and not have to use apt to get it with setting proxy.